components/ContributionConfirmationModal.js

```diff
@@ -67,15 +69,18 @@ const ContributionConfirmationModal = ({ order, onClose }) => {
     };
 
     if (amount !== order.amount.valueInCents) {
-      orderUpdate.amount = { valueInCents: amount, currency };
+      orderUpdate.amount = { valueInCents: Math.round(amount / hostCurrencyFxRate), currency: orderCurrency };
```

Comment:
Sorry, we don't want this. If the Host admin input an amount, it needs to be recorded "as is" and not through conversions.

Reply:
Here the issue is that on the API side we need to send things in `orderCurrency`; as that's what we will be recording in the `order` table. If we try to send it using `hostCurrency`, then this same conversion will need to be done on the API side. Is this okay? 🤔